storybook/src/components/Breakout/Breakout.stories.tsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { AspectRatio, Breakout, Image, Paragraph, Screen, Spotlight } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+import { exampleParagraph } from '../shared/exampleContent'
+
+const paragraph = exampleParagraph()
+
+const meta = {
+  title: 'Components/Layout/Breakout',
+  component: Breakout,
+  decorators: [
+    (Story) => (
+      <Screen>
+        <Story />
+      </Screen>
+    ),
+  ],
+} satisfies Meta<typeof Breakout>
+
+export default meta
+
+const cellMeta = {
```

Comment:
These controls do not seem to be working. `colStart`, `rowSpan` and `rowStart` aren't shown, and only `colSpan` `all` has the desired effect, using a number there doesn't work.